lib/managers/webhooks.js

```diff
@@ -193,6 +193,120 @@ Webhooks.prototype.delete = function(webhookID, callback) {
 	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
 };
 
+const crypto = require('crypto');
+
+/**
+ * Compute the message signature (see https://docs.box.com/reference#signatures)
+ *
+ * @param {string} body - The request body of the webhook message
+ * @param {Object} headers - The request headers of the webhook message
+ * @param {string} signatureKey - The signature to verify the message with
+ * @returns {string} - The message signature
+ * @private
+ */
+function computeSignature(body, headers, signatureKey) {
+	if (!signatureKey) {
+		return undefined;
+	}
+
+	if (headers['box-signature-version'] !== '1') {
+		// console.log('Unrecognized signature version: %s', headers['box-signature-version']);
```

Comment:
I thought that they were useful to leave in.  If someone wants to extend or debug this code it will be much easier if they can comment these back in during development.  Is there a better way to do that?